hostscripts/soc-ci/soc-ci

```diff
@@ -54,28 +58,28 @@ def _config_get():
     """get the configuration"""
     conf = configparser.RawConfigParser()
     conf.read([USER_CONFIG_PATH])
-    section_name = "defaults"
-    if not conf.has_section(section_name):
+    CONFIG_SECTION = "defaults"
```

Comment:
Duplicated from above? I see USER_CONFIG_PATH is defined just above it, and not declared in its use in _config_get().

Reply:
Done